File: c#/web/WebApplication1/Entity/UploadFilesResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Entity
{
    /// <summary>
    /// 上传文件的返回值
    /// </summary>
    public class UploadFilesResult
    {
        /// <summary>
        /// 文件扩展名
        /// </summary>
        public string extension { set; get; }

        /// <summary>
        /// 文件的sha1
        /// </summary>
        public string sha1 { set; get; }
    }
}

File: java/common/src/main/java/com/txj/common/entity/UploadFilesResult.java
package com.txj.common.entity;

/**
 * 上传文件的返回值
 * @author admin
 *
 */
public class UploadFilesResult {
	/**
	 * 文件扩展名
	 */
	private String extension;
	/**
	 * 文件的sha1
	 */
	private String sha1;
	
	public String getExtension() {
		return extension;
	}
	public void setExtension(String extension) {
		this.extension = extension;
	}
	public String getSha1() {
		return sha1;
	}
	public void setSha1(String sha1) {
		this.sha1 = sha1;
	}
}
